tests/environment/files/test-users.csv

```diff
@@ -0,0 +1,5 @@
+email,first_name,last_name,preferred_username,phone,enabled
+john.doe@example.com,John,Doe,jdoe,"+1111111111",true
+jane.smith@example.com,Jane,Smith,jasmith,"+2222222222",true
+alice.johnson@example.com,Alice,Johnson,ajohnson,,true
+bob.williams@example.com,Bob,Williams,bwilliams,"+4444444444",true
```

Comment:
Nit: Another EOF newline ;) The precommit hook checks for this, maybe we should add a check to CI as well.

Reply:
Done.